src/cascadia/TerminalApp/defaults.json

```diff
@@ -45,8 +45,7 @@
             "icon": "ms-appx:///ProfileIcons/{0caa0dad-35be-5f56-a8ff-afceeeaa6101}.png",
             "padding": "8, 8, 8, 8",
             "snapOnInput": true,
-            "useAcrylic": true,
-            "acrylicOpacity": 0.75
+            "useAcrylic": false

```

Comment:
```suggestion
            "useAcrylic": false,
            "acrylicOpacity": 0.75
```

Reply:
Won't-fix: Acrylic is disabled and thus (like in PowerShell's default profile), opacity is also omitted.